src/pages/Classes/AddClasses.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component, PropTypes } from 'react'
+import { Container, Header, Divider, Dropdown } from 'semantic-ui-react'
+
+export default class AddClasses extends Component {
+  static courseToProjects = {
```

Comment:
can we put all this in an outside `.stub` file and import it? This will get really confusing once we get api calls